channel.py

```diff
@@ -205,6 +205,7 @@ def test_connection(cls, channels):
         """
         try:
             channel, = channels
+            Transaction().set_context(magento_channel=channel.id)
```

Comment:
this method should be removed from here and functionality should be moved to `sale.channel` as configuration wizard